e2e-tests/functions

```diff
@@ -9,7 +9,7 @@ API="psmdb.percona.com/v1-6-0"
 IMAGE=${IMAGE:-"perconalab/percona-server-mongodb-operator:${GIT_BRANCH}"}
 IMAGE_PMM=${IMAGE_PMM:-"perconalab/percona-server-mongodb-operator:master-pmm"}
 IMAGE_MONGOD=${IMAGE_MONGOD:-"perconalab/percona-server-mongodb-operator:master-mongod4.2"}
-IMAGE_BACKUP=${IMAGE_BACKUP:-"perconalab/percona-server-mongodb-operator:master-backup"}
+IMAGE_BACKUP=${IMAGE_BACKUP:-"perconalab/percona-server-mongodb-operator:sharded-backup"}
```

Comment:
probably we should not change this, also master-backup was rebuilt from PBM main branch so I guess it supports this

Reply:
https://github.com/percona/percona-server-mongodb-operator/pull/455/commits/89d4d3c5c3825386d9e410dcc3db481860870901